scripts/tests/chiptest/accessories.py

```diff
@@ -116,3 +122,11 @@ def __startXMLRPCServer(self):
 
     def __stopXMLRPCServer(self):
         self.server.shutdown()
+
+    def __createCommandLineOptions(self, args):
+        # args should contain a list of strings in key-value pair, e.g. [option1, value1, option2, value2, ...]
+        options = {}
+        if (len(args) % 2) == 0:
```

Comment:
what if length is odd? Should we throw an exception?

Reply:
I figured if improper args were passed in, we just ignore it altogether. Without args passed in, the default values of the application being launched will kick in. Seems a bit extreme to throw an exception rather than letting it continue to run?